addons/udes_stock_cron/models/stock_move.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+from odoo import models, api, _
+
+import logging
+_logger = logging.getLogger(__name__)
+
+# Map move state to the refactor stage.
```

Comment:
We shouldn't be migrating any refactoring logic as part of this story. We'll have future stories to handle this, as we used a different approach in v13 which we'll want to use in v14.